### PR TITLE
setup project configuration

### PR DESCRIPTION
**Pull request by eliaset096**
Review initial setup project configuration

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,6 +12,10 @@ cypress/results/*
 cypress/reports/*
 cypress/screenshots/*
 cypress/videos/*
+...
```
Comment (holgiosalos) [inline .gitignore]: these points were in the guide as a reference that there is more content below and above the line that needs to be added. Please remove them :)

File: .gitignore [hunk 2]
```diff
@@ -12,6 +12,10 @@ cypress/results/*
...
+cypress/integration/examples/*
+...
```
Comment (holgiosalos) [inline .gitignore]: these points were in the guide as a reference that there is more content below and above the line that needs to be added. Please remove them :)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "type": "git",
+    "url": "git+https://github.com/eliaset096/cypress-training.git"
+  },
+  "keywords": [
+    "ui-testing",
+    "dojo",
+    "practice",
+    "cypress"
+  ],
+  "author": "Elías Estupiñan eliaset096@gmail.com eliaset096\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D\u001b[D  e\u001b[D\u001b[D\u001b[D \u001b[D\u001b[C \u001b[D\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[C\u001b[Cn  eliaset096@gmail.com  eliaset096",
```
Comment (holgiosalos) [inline package.json]: please cleanup the special characters

File: .gitignore (hunk 1 above)
Comment (eliaset096) [reply .gitignore]: Points deleted.

File: .gitignore (hunk 2 above)
Comment (eliaset096) [reply .gitignore]: Points deleted.

File: package.json (hunk 3 above)
Comment (eliaset096) [reply package.json]: Special characters removed.

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**